specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -3763,6 +3831,10 @@
           "description": "An ODBC connection string. Type: string, SecureString or AzureKeyVaultSecretReference.",
           "type": "object"
         },
+        "pwd": {
```

Comment:
Why is this one named pwd while others password?

Reply:
The property name needs to be same with the name in connection string. In this Netezza linked service case, a typical connection string is `Server=<server>;Port=<port>;Database=<database>;UID=<user name>;PWD=<password>`.